module4/lessons/Week1/STARInterviewTechnique.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: page
+title: STAR Interview Technique
+---
+
+## Learning Goals
+* Understand how to use the STAR technique when answering interview questions
+* Begin to gather a collection of stories to use in a behavioral interview
+
+### Preparation
+Before this session, review the three strengths you identified in the "Your Strengths" PD session.
+
+### Warmup

```

Comment:
Love that this warmup puts the students in the interviewer's seat!

Reply:
Agree! Pulled this question from a FE/BE PD, really like the putting them in the other seat idea.